R/fit_uniform.R

```diff
@@ -114,12 +114,12 @@ fit_uniform_helper <- function(
   }
 
   return(
-    list(
-      "par" = NULL,
-      "dist" = "unif",
-      "metric" = metric,
-      "value" = value,
-      "dens" = function(x = NULL, mpar = NULL, scale = TRUE) {
+    new_ModelFit(
+      par = NULL,
+      distribution = "unif",
+      metric = metric,
+      value = value,
+      density_function = function(x = NULL, mpar = NULL, scale = TRUE) {
```

Comment:
It looks like there is a mix of `dens` and `density_function`. They should be standardized across all functions:

https://github.com/uclahs-cds/public-R-HistogramZoo/blob/helenzhu-model_fit_s3_class/R/create_coverageplot.R#L338

https://github.com/uclahs-cds/public-R-HistogramZoo/blob/helenzhu-model_fit_s3_class/R/create_residualplot.R#L104

(I think there are a few more, I just did a "Find-all" for `dens(` to see where the function is called.. I think I am missing where `dens` is actually assigned.)